pom.xml

```diff
@@ -1364,6 +1364,38 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>oss</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-gpg-plugin</artifactId>
+            <version>1.5</version>
```

Comment:
- [ ] available internally?

Reply:
Nope :(
only 3.1.0. I will test if that works for us